File: BankAPI/Models/Role.cs
using System;
using System.Collections.Generic;

namespace BankAPI.Models
{
    public partial class Role
    {
        public int IdRole { get; set; }
        public string NameRole { get; set; } = null!;
        public bool? RoleDeleted { get; set; }
    }
}

File: app/src/main/java/com/example/bankandroid/Models/Role.java
package com.example.bankandroid.Models;

public class Role {
    public int idRole;
    public String nameRole;
    public boolean roleDeleted;

    public Role(int idRole, String nameRole, boolean roleDeleted) {
        this.idRole = idRole;
        this.nameRole = nameRole;
        this.roleDeleted = roleDeleted;
    }

    public int getIdRole() {
        return idRole;
    }

    public void setIdRole(int idRole) {
        this.idRole = idRole;
    }

    public String getNameRole() {
        return nameRole;
    }

    public void setNameRole(String nameRole) {
        this.nameRole = nameRole;
    }

    public boolean isRoleDeleted() {
        return roleDeleted;
    }

    public void setRoleDeleted(boolean roleDeleted) {
        this.roleDeleted = roleDeleted;
    }
}
